src/v/kafka/server/handlers/alter_partition_reassignments.cc

```diff
@@ -171,9 +171,8 @@ partitions_request_iterator validate_partitions(
               return true;
           }
 
-          if (
-            !tp_metadata.has_value()
-            || !tp_metadata.value().is_topic_replicable()) {
+          if (!tp_metadata.has_value()) {
+              vlog(klog.debug, "Topic metadata is undefined");
               return false;
           }
 
```

Comment:
Since you're adding logging to check if `tp_metadata` has value, do you also want to add any logging below to printout the replication factor that `tp_metadata` contains?  Or is it printed out anywhere and I'm blind and don't see it :)

Reply:
I thought the same thing but if this check fails because the incoming request doesn't match the topic replication factor, then we know the exact source of the problem. So a log may just be extra noise.

Reply:
Actually, I'll put one in to be safe.